### PR TITLE
update user buckets;

### PR DESCRIPTION
**Pull request by sravfeyn**
https://docs.google.com/document/d/1aDHbjhpdWwvgKjlOY4f5ksOVQWAv3CwyMep2mDv09uY/edit

@esoergel 

@nickpell Is this all that's needed for that spec?

### CODE REVIEW

File: corehq/apps/accounting/bootstrap/config/user_buckets_jan_2017.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+from decimal import Decimal
+
+from corehq.apps.accounting.models import (
+    FeatureType,
+    SoftwarePlanEdition,
+)
+
+BOOTSTRAP_CONFIG = {
+    (SoftwarePlanEdition.COMMUNITY, False): {
+        'role': 'community_plan_v0',
```
Comment (nickpell) [inline corehq/apps/accounting/bootstrap/config/user_buckets_jan_2017.py]: should be ```community_plan_v1```

**Review by nickpell — CHANGES_REQUESTED** [review]
one comment; otherwise looks good

File: corehq/apps/accounting/bootstrap/config/user_buckets_jan_2017.py (hunk 1 above)
Comment (sravfeyn) [inline corehq/apps/accounting/bootstrap/config/user_buckets_jan_2017.py]: (Updated) @nickpell

**Comment by nickpell** [comment]
👍

**Pull request merged**